SR2023/2023-02-13-virtual-competition-code-submission.md

```diff
@@ -0,0 +1,35 @@
+---
+to: Student Robotics 2023
+subject: Submit your robot for the SR2023 Virtual Competition
+---
+
+Hi
+
+You can now submit your robot code through our website for the [virtual competition](https://studentrobotics.org/events/sr2023/virtual-competition/).
+
+[Submit your robot code](https://studentrobotics.org/code-submitter/).
+
+You have until 8pm (UK) Friday 24th February to submit your team's latest code. You can upload early, but at that time we'll capture all the uploads. If you fail to upload any code, you will be marked as absent from all the matches and score 0 points. This will not affect your ability to compete in the [final competition](https://studentrobotics.org/events/sr2023/competition/).
```

Comment:
Nit
```suggestion
You have until 8pm (UK Time) Friday 24th February to submit your team's latest code. You can upload early, but at that time we'll capture all the uploads. If you fail to upload any code, you will be marked as absent from all the matches and score 0 points. This will not affect your ability to compete in the [final competition](https://studentrobotics.org/events/sr2023/competition/).
```